ejercicio1.rb

```diff
@@ -22,8 +21,6 @@
   end
 
   uri = URI.parse('https://api.github.com/gists')
```

Comment:
Maybe you should save the link to a constant because the link should not change.